dandi/cli/cmd_organize.py

```diff
@@ -214,6 +239,36 @@ def _get_metadata(path):
 
     metadata = create_unique_filenames_from_metadata(metadata)
 
+    # update metadata with external_file information:
+    external_files_missing_in_nwbfiles = [
+        len(m["external_file_objects"]) == 0 for m in metadata
+    ]
+
+    if all(external_files_missing_in_nwbfiles) and modify_external_file_fields:
+        lgr.warning(
+            "modify_external_file_fields specified but no external_files found "
+            "linked to any nwbfile found in %s",
+            paths,
+        )
+    elif (
+        not all(external_files_missing_in_nwbfiles) and not modify_external_file_fields
+    ):
+        raise ValueError(
+            "modify_external_file_fields option not specified but found "
+            "external video files linked to the nwbfiles "
+            f"""{[metadata[no]['path']
```

Comment:
Join the elements of the list together with `", "` instead of stringifying a list.

Reply:
@Saksham20 You missed this comment.

Reply:
thanks for catching that: https://github.com/dandi/dandi-cli/pull/841/commits/d1a71e584e6be75c38b8e2dfbae93a5497c3a604